.github/workflows/test_install.yml

```diff
@@ -0,0 +1,51 @@
+on: [push]
+
+jobs:
+  installer_test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        verifier-release: [main, v0.0.1, v1.0.0, v1.0.1, v1.0.2, v1.1.0, v1.1.1, v1.2.0, v1.3.0]
```

Comment:
this is going to get out of sync quickly. Any ideas?
One idea I have is to list the versions available for the repo, and create the matrix dynamically. You can check online that this is possible - I think it makes use of script injections.. but it works :-)

Reply:
Looks like there's an API for listing releases: https://docs.github.com/en/rest/releases/releases#list-releases